pycyphal/transport/commons/high_overhead_transport/_transfer_serializer.py

```diff
@@ -62,9 +62,11 @@ def serialize_transfer(
     payload_length = sum(map(len, fragmented_payload))
     if payload_length <= max_frame_payload_bytes:  # SINGLE-FRAME TRANSFER
```

Comment:
Since the transfer CRC is now included in single-frame transfers also, this condition needs to take the size of the CRC into account. Can you please also add a doctest for this case?